app/javascript/chat/chat.jsx

```diff
@@ -1409,7 +1410,35 @@ export default class Chat extends Component {
       .getElementById('jumpback_button')
       .classList.remove('chatchanneljumpback__hide');
   };
-
+  handleImageDrop = (e) => {
+    document.getElementById('messagelist').style.opacity = 1;
+    dragDrop(e, this.handleImageSuccess, this.handleImageFailure);
+  };
+  handleImageSuccess = (res) => {
+    const mLink = `![alt text](${res.links[0]})`;
```

Comment:
Consider a complete name as this variable will get minified by Terser in webpack. I'm assuming `mLink` means markdown link?

Reply:
Yes, markdown link.